README.md

```diff
@@ -10,36 +10,42 @@
 
 
 ## Functionalities
-- [ ]  < Standalone Script for parsing csv file, generating passwords and uploading it to database >
-- [ ]  < insert functionality >
-- [ ]  < insert functionality >
-- [ ]  < insert functionality >
+- Standalone Script for parsing csv file, generating passwords and uploading it to database
+- A login and registration route
 
 <br>
 
 
 ## Instructions to run
 
 * Pre-requisites:
-	-  < insert pre-requisite >
-	-  < insert pre-requisite >
+	-  Node.js installed with npm enabled
+	-  An active internet connection to connect to the cloud database
+	-  All the npm packages mentioned in the package.json
 
-* < directions to install >
+* To setup the environment and install packages
 ```bash
-< insert code >
+npm init
+npm install @hapi/joi bcryptjs body-parser csv-parser csv-writer dotenv express mongoose nodemon fs uuid4 jsonwebtoken
```

Comment:
No need. `package.json` has a list of packages to install. Just write `npm install`